lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -212,61 +212,94 @@
   }
 }
 
-// +Dashboard - Course Listing
-// ====================
-.dashboard {
-  .my-courses {
+.xseries-epmty-section{
     @include float(left);
-    margin: 0;
-    width: flex-grid(9);
-
-
-    .wrapper-header-courses {
-      border-bottom: 4px solid $border-color-l4;
-      margin-bottom: $baseline;
-
-      .header-courses {
-        @extend %t-title5;
-        @include padding-right($baseline/2);
-      }
-    }
-
-    // CASE: empty dashboard
+    @include outer-container;
+    @include span-columns(9);
+}
+// CASE: empty dashboard
     .empty-dashboard-message {
+      border: 3px solid $gray-l4;
+      background: $gray-l6;
       padding: ($baseline*2) 0;
       text-align: center;
 
       p {
+        @include font-size(24);
         color: $lighter-base-font-color;
-        font-style: italic;
         margin-bottom: $baseline;
         text-shadow: 0 1px rgba(255,255,255, 0.6);
       }
 
       a {
-        background: rgb(240,240,240);
-        @include background-image($button-bg-image);
-        background-color: $button-bg-color;
-        border: 1px solid $border-color-2;
-        border-radius: 4px;
+        background-color: $blue;
+        border: 1px solid $blue;
         box-shadow: 0 1px 8px 0 $shadow-l1;
         @include box-sizing(border-box);
-        color: $base-font-color;
+        color: $white;
         font-family: $sans-serif;
         display: inline-block;
         letter-spacing: 1px;
-        @include margin-left($baseline/4);
-        padding: 5px 10px;
-        text-shadow: 0 1px rgba(255,255,255, 0.6);
+        margin-top: ($baseline/4);
+        margin-left: ($baseline/4);
+        padding: 15px 20px;
 
         &:hover, &:focus {
-          color: $link-color;
+          background: $blue-l2;
           text-decoration: none;
         }
       }
+
+      .find-xseries-programs {
```

Comment:
@tasawernawaz I see your point about reusability, however the intention is to move these pages to the new edX Pattern Library views soon so it would be great to keep these styles in a new partial to make it easier to move across.